api/extension-guides/chat.md

```diff
@@ -267,7 +268,59 @@ cat.followupProvider = {
 };
 ```
 
-> **Tip:** Follow-ups should be written as questions or directions, not just concise commands.
+> [!TIP]
+> Follow-ups should be written as questions or directions, not just concise commands.
+
+### Implement participant detection
+
+To make it easier to use chat participants with natural language, you can implement participant detection. Participant detection is a way to automatically route the user's question to a suitable participant, without having to explicitly mention the participant in the prompt. For example, if the user asks "How do I add a login page to my project?", the question would be automatically routed to the `@workspace` participant.
```

Comment:
"... routed to the `@workspace` participant because it can answer questions about the user's project"